sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/ParserIntegrationSamples.cs

```diff
@@ -13,16 +13,22 @@ internal class ParserIntegrationSamples
     {
         public static async Task GetModelsAndParseAsync()
         {
+            #region Snippet:ModelsRepositorySamplesParserIntegrationGetModelsAndParseAsync
+
             var dtmi = "dtmi:com:example:TemperatureController;1";
             var client = new ModelsRepositoryClient();
             IDictionary<string, string> models = await client.GetModelsAsync(dtmi).ConfigureAwait(false);
             var parser = new ModelParser();
             IReadOnlyDictionary<Dtmi, DTEntityInfo> parseResult = await parser.ParseAsync(models.Values.ToArray());
             Console.WriteLine($"{dtmi} resolved in {models.Count} interfaces with {parseResult.Count} entities.");
+
+            #endregion Snippet:ModelsRepositorySamplesParserIntegrationGetModelsAndParseAsync
         }
 
         public static async Task ParseAndGetModelsWithExtensionAsync()
         {
+            #region Snippet:ModelsRepositorySamplesParserIntegrationParseAndGetModelsAsync
+
             var dtmi = "dtmi:com:example:TemperatureController;1";
```

Comment:
Here's another case of declaring a variable before it is needed. It should come after the client init.

Reply:
Fixed here and one more instance.